test/dotnet/scenarios.json

```diff
@@ -1,84 +1,58 @@
 {
-    "install_additional_dotnet": {
-        "image": "ubuntu:focal",
-        "features": {
-            "dotnet": {
-                "version": "6.0.301",
-                "additionalVersions": "5.0,3.1.420"
-            }
-        }
-    },
-    "install_dotnet_3": {
-        "image": "ubuntu:focal",
-        "features": {
-            "dotnet": {
-                "version": "3"
-            }
-        }
-    },
-    "install_dotnet_6_bullseye": {
-        "image": "debian:bullseye",
-        "features": {
-            "dotnet": {
-                "version": "6"
-            }
-        }
-    },
-    "install_dotnet_6_focal": {
-        "image": "ubuntu:focal",
-        "features": {
-            "dotnet": {
-                "version": "6"
-            }
-        }
-    },
-    "install_dotnet_7_bullseye": {
-        "image": "debian:bullseye",
+    "install_dotnet_lts": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "7"
+                "version": "lts"
             }
         }
     },
-    "install_dotnet_7_jammy": {
-        "image": "ubuntu:jammy",
+    "install_dotnet_specific_release": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu-20.04",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "7"
+                "version": "3.1"
             }
         }
     },
-    "install_dotnet_6_jammy": {
-        "image": "mcr.microsoft.com/devcontainers/base:jammy",
+    "install_dotnet_specific_release_and_feature_band": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu-20.04",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "6"
+                "version": "5.0.3xx"
             }
         }
     },
-    "install_dotnet_latest": {
-        "image": "ubuntu:jammy",
+    "install_dotnet_exact_version": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "latest"
+                "version": "8.0.100-preview.6.23330.14"
             }
         }
     },
-    "install_dotnet_lts": {
-        "image": "ubuntu:focal",
+    "install_dotnet_multiple_versions": {
+        "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+        "remoteUser": "vscode",
         "features": {
             "dotnet": {
-                "version": "lts"
+                "version": "8.0.100-preview.6.23330.14",
+                "additionalVersions": [
```

Comment:
How does this test pass when `additionalVersions` should have been a comma-separated string instead of an array?

Reply:
Cool, there is an automatic conversion somewhere from array to comma-separated strings.